### ISSUE TITLE
lungojs is not working in ipad it is showing black screen but in i phone it is working perfectly . 

### ISSUE THREAD

**Issue by gaurav1992** [tapquo/Lungo.js#214]
i used the lungoJS v2.1 but i have again same problem .Is that style sheet problem or java script ?

**Comment by maxaud** [comment]
I too am seeing some irregularities using an iPad.
Asides never close.
Scrolls and carousels don't swipe.
Multiple pages show at a time.
See screenshot:

![lungo-screenshot](https://f.cloud.github.com/assets/2069741/823133/a26681a8-f002-11e2-93ad-ce056ed3e002.PNG)

**Comment by maxaud** [comment]
It should be noted that my screenshot is from 2.2.

**Comment by soyjavi** [comment]
I'll check ... sorry for the inconvenience

**Comment by ghost** [comment]
Any update on this soyjavi?

**Comment by tpae** [comment]
yes.. it looks terrible on iPad, other than that.. i love Lungo! It has the potential to be the best, just needs to be maintained...

**Comment by gauravdhiman** [comment]
@soyjavi - Any update on this issue ?
I tried Lungo 2.2.1 on iPad and I too faced distorted UI. Header bar was shown in footer, aside was not shown and rest space was black.

Can you kindly look into this issue and fix it, it will help many developers aiming iPad with Lungo.
